neuralprophet/configure.py

```diff
@@ -263,7 +263,9 @@ def __post_init__(self):
             self.n_changepoints = len(self.changepoints)
             self.changepoints = pd.to_datetime(self.changepoints).sort_values().values
 
-        if type(self.trend_reg_threshold) == bool:
+        if self.trend_reg_threshold is None:
+            pass
+        elif type(self.trend_reg_threshold) == bool:
```

Comment:
shouldn't this be `isinstance(..., bool)`?

Reply:
good point, did not change it to my defense, haha ;)

Reply:
fixed